diff --git a/rpm/harbour-cutespotify.spec b/rpm/harbour-cutespotify.spec
index fd8260f..9e3d8e6 100644
--- a/rpm/harbour-cutespotify.spec
+++ b/rpm/harbour-cutespotify.spec
@@ -17,7 +17,7 @@ Name:       harbour-cutespotify
 %{?qtc_builddir:%define _builddir %qtc_builddir}
 Summary:    CuteSpotify
 Version:    1.5.2
-Release:    1
+Release:    0
 Group:      Qt/Qt
 License:    BSD
 Source0:    %{name}-%{version}.tar.bz2
